src/components/ToDoContainer/index.js

```diff
@@ -1,9 +1,143 @@
 import React, { Component } from "react";
 
+
+let id = 0;
```

Comment:
Do not declare global variables - you can place `id` in the state of one of your components and update it along with the todos whenever you add a new todo